spec/PeriodicBackgroundSync-index.bs

```diff
@@ -165,11 +192,13 @@ Algorithms {#algorithms}
     When the user agent starts, run the following steps [=in parallel=]:
     1. While true:
       1. Wait for [=minimum periodic sync interval across origins=].
+        Note: This can be used to mitigate concerns raised in [[#security]].
+
       1. Let |firedPeriodicSync| be false.
       1. While |firedPeriodicSync| is false:
         1. Wait for some user agent defined amount of time.
 
-          Note: This can be used to group synchronization for different [=periodic sync registrations=] into a single device wake-up.
+          Note: This can be used to group synchronization for different [=periodic sync registrations=] into a single device wake-up. This can be used to mitigate the concerns raised in [[#security]].
```

Comment:
```suggestion
          Note: This can be used to group synchronization for different [=periodic sync registrations=] into a single device wake-up. This can also be used to mitigate the concerns raised in [[#security]].
```

since the two uses seem independent.

Reply:
Did you mean to merge this @mugdhalakhani?

Reply:
yes! It got over-written, I'll fix.